app/views/undeliverable_email.scala.html

```diff
@@ -0,0 +1,55 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
```

Comment:
would be better to move this file in a separate package views.email, if you find it time consuming then leave  it.